### ISSUE TITLE
ModuleNotFoundError: No module named 'functions'

### ISSUE THREAD

**Issue by Lamm-a** [nselem/bigscape#1]
When trying to call the `bigscape.py` script with the -h flag I get this error:
`ModuleNotFoundError: No module named 'functions'`

Any idea how to fix this?

**Comment by jorgecnavarrom** [comment]
Hi Lamm-a
Just to be sure: are you running the docker version of BiG-SCAPE, or did you download the code from the repository (if it's the latter, it sounds as if you only had the bigscape.py file)

**Comment by Lamm-a** [comment]
I downloaded all the dependencies with the conda yaml file then cloned the bigscape.py script. Do I need to clone the whole repository even if in conda?

**Comment by jorgecnavarrom** [comment]
Yes, you need the rest of the files as the conda items only provide the required libraries to run BiG-SCAPE. Could you clone the whole repository and let me know if that solved the problem?

**Comment by Lamm-a** [comment]
Are the inout files meant to be all antismash .gbk outputs, such as <sample_name>.clusterXXX.gbk orjust the <sample_name>.final.gbk files?

**Comment by jorgecnavarrom** [comment]
We only expect the clusterXXX files (in fact, BiG-SCAPE will ignore all files containing "final" in their name). Additionally, make sure all BGC file names are different!

**Comment by Lamm-a** [comment]
By BGC files do you mean samples or?

**Comment by jorgecnavarrom** [comment]
yes, or genome name. In any case, just to be careful because, depending on the original genome data, you could end up with different folders (corresponding to genomes or metagenome samples) having names like "scaffold1.cluster1.gbk", and when reading your input, BiG-SCAPE will see two (or more) files with the same name and will ignore all except for the first one.

**Comment by Lamm-a** [comment]
Okay thank you, this worked. I am get very minimal networking through. I assume this then means there is little similarity between the sequences? I am running this on a group of actinobacteria spanning several species, having very closly related isolates reduce the ability of network formating between BGCs from more distance isolates?

**Comment by dominicOT** [comment]
I am trying to import 'functions.online' library to make my voice assistant work online but its not working. Any idea how to fix it?

**Comment by jorgecnavarrom** [comment]
@dominicOT In the original question at the beginning of this thread, the issue was that there was a missing file called `functions.py`. The main program was trying to use it but couldn't find it and that's why the "ModuleNotFoundError: No module named 'functions'" error appeared. But I don't know what is happening in your case, sorry

**Comment by dominicOT** [comment]
@jorgecnavarrom it’s quite the same issue

**Comment by dominicOT** [comment]
@jorgecnavarrom it’s quite the same issue
